velox/docs/functions/spark/regexp.rst

```diff
@@ -34,3 +34,44 @@ See https://github.com/google/re2/wiki/Syntax for more information.
     the entire string by anchoring the pattern using ``^`` and ``$``. ::
 
         SELECT rlike('1a 2b 14m', '\d+b'); -- true
+
+.. spark:function:: regex_replace(string, pattern, overwrite) -> varchar
+
+    Replaces all substrings in ``string`` that match the regular expression ``pattern`` with the string ``overwrite``. If no match is found, the original string is returned as is.
+    There is a limit to the number of unique regexes to be compiled per function call (kMaxCompiledRegexes defined in velox/functions/lib/Re2Functions.h).
```

Comment:
It would be inconvenient for the reader to search the code for kMaxCompiledRegexes. Let's hard-code the number here.

Reply:
Fixed